code/modules/mob/living/silicon/ai/latejoin.dm

```diff
@@ -13,7 +13,7 @@ GLOBAL_LIST_EMPTY(empty_playable_ai_cores)
 
 /mob/living/silicon/ai/proc/cryo_AI()
 	GLOB.empty_playable_ai_cores += new /obj/structure/AIcore/deactivated(loc)
-	GLOB.global_announcer.autosay("[src] has been moved to intelligence storage.", "Artificial Intelligence Oversight")
+	GLOB.global_announcer.autosay("[src] has been moved to intelligence storage.", "Artificial Intelligence Oversight", follow_target_override = src)
```

Comment:
I think this should follow the new AIcore/deactivated since src gets deleted later in this proc

Reply:
Addressed, good catch!